### PR TITLE
Feat/tag list

### PR DESCRIPTION
**Pull request by Frankwarnaar**
## What changed

- Refactor tag-item into tag-list, so we can reuse the entire list component instead of just the item
- Add tag list to landing page
- Add default margins to titles in structured text

## How to test

Can be tested on https://deploy-preview-639--voorhoede-website.netlify.app/en/lustrum-new/?preview=true&previewSecret=blue-unsalted-ravioli

### CODE REVIEW

File: migrations/1683712628_tagList.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { Client, SimpleSchemaTypes } from "@datocms/cli/lib/cma-client-node";
+
+export default async function (client: Client) {
+  const newFields: Record<string, SimpleSchemaTypes.Field> = {};
+  const newItemTypes: Record<string, SimpleSchemaTypes.ItemType> = {};
+
+  console.log("Create new models/block models");
+
+  console.log('Create block model "Tag list" (`tag_list`)');
```
Comment (lsarni) [inline migrations/1683712628_tagList.ts:9]: We don't want to keep using naming format `Structured Text -` to identify that this is a structured text block?

File: src/pages/[language]/[slug]/index.vue [hunk 2]
```diff
@@ -63,6 +63,11 @@
         :title="section.title"
         :items="section.items"
       />
+      <image-card-grid
+        v-if="section.__typename === 'SectionImageCardGridRecord'"
+        :title="section.title"
+        :items="section.items"
+      />
```
Comment (lsarni) [inline src/pages/[language]/[slug]/index.vue:70]: Duplicated from what was already merged

Comment (Frankwarnaar) [reply src/pages/[language]/[slug]/index.vue:70]: Whoops, my bad

File: migrations/1683712628_tagList.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import { Client, SimpleSchemaTypes } from "@datocms/cli/lib/cma-client-node";
+
+export default async function (client: Client) {
+  const newFields: Record<string, SimpleSchemaTypes.Field> = {};
+  const newItemTypes: Record<string, SimpleSchemaTypes.ItemType> = {};
+
+  console.log("Create new models/block models");
+
+  console.log('Create block model "Structured text - Structured text - Tag list" (`tag_list`)');
```
Comment (lsarni) [inline migrations/1683712628_tagList.ts:9]: `Structured text` is written twice, and `tag_list` should be `structured_text_tag_list`

File: src/pages/[language]/lustrum/index.vue [hunk 4]
```diff
@@ -18,14 +18,7 @@
...
       </h2>
-      <ul class="page-lustrum__facts-list">
-        <tag-item
-          v-for="fact in data.page.factsList"
-          :key="fact.id"
-          :amount="fact.amount"
-          :label="fact.label"
-        />
-      </ul>
+      <tag-list :items="data.page.factsList" />
```
Comment (lsarni) [inline src/pages/[language]/lustrum/index.vue:21]: You should also delete the css for `.page-lustrum__facts-list`

**Review by lsarni — APPROVED** [review]

File: datocms.config.json [hunk 5]
```diff
@@ -8,4 +8,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```
Comment (Siilwyn) [inline datocms.config.json:11]: Awesome 🍡

File: src/components/tag-list/tag-list.vue [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+<style>
+  .tag-list {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    gap: var(--spacing-smaller);
+  }
+
+  .tag-list__item {
+    padding: .2em 1em .1em 1em;
```
Comment (Siilwyn) [inline src/components/tag-list/tag-list.vue:34]: My gut feeling says this should be `rem` or is there a specific reason for `em` here?

**Review by Siilwyn — APPROVED** [review]

**Pull request merged**